.github/workflows/shortint_benchmark.yml

```diff
@@ -0,0 +1,122 @@
+# Run shortint benchmarks on an AWS instance and return parsed results to Slab CI bot.
```

Comment:
Probably the same comment as above apply

Reply:
I don't understand your point there.

Reply:
sorry I meant, all boolean comment will apply in this file

Reply:
Oh yes obviously !